lizard_auth_client/client.py

```diff
@@ -713,7 +713,7 @@ def get_billable_organisation(user, role_code=models.Role.BILLING_ROLE_CODE):
                 synced = True
 
 
-def sso_server_url(name):
+def sso_server_url(name, use_cache=True):
```

Comment:
Die use_cache wordt nergens (meer) aangeroepen. Was die er alleen voor een test die nu anders is opgezet?
Ik gok dat 'ie nu weg kan?